ceph.spec.in

```diff
@@ -131,6 +131,7 @@ BuildRequires:	libcurl-devel
 BuildRequires:	libudev-devel
 BuildRequires:	libtool
 BuildRequires:	libxml2-devel
+BuildRequires:	lz4-devel
```

Comment:
In openSUSE/SLE the package really is called `liblz4-devel` as @theanalyst said, and according to http://rpm.pbone.net it's called `lz4-devel` in RHEL/CentOS/Fedora. So this line should be moved down to the "distro-conditional dependencies" section and split into two.